sympy/sets/tests/test_sets.py

```diff
@@ -1744,3 +1744,10 @@ def test_issue_14336():
     x = Symbol("x")
     U -= U.intersect(Ne(x, 1).as_set())
     U -= U.intersect(S.true.as_set())
+
+def test_issue_9855():
+    #https://github.com/sympy/sympy/issues/9855
+    x, y, z = symbols('x, y, z', real=True)
+    s1 = Interval(1,x) & Interval(y,2)
+    s2 = Interval(1,2)
```

Comment:
```suggestion
    s1 = Interval(1, x) & Interval(y, 2)
    s2 = Interval(1, 2)
```

Reply:
Sorry I did not see the comment yesterday. Thank you.